File: python/problems_01_25/tests/problem_02_test.py
from problems_01_25.problem_02 import even_fibsum
import unittest

class Problem02Test(unittest.TestCase):
    """
    test the implementation of Project Euler Problem #2
    """

    def test_terms_less_than_20(self):
        """
        test problem 2 case were limit is 20

        there are 2 terms (2,8) less than 20
        in the Fibbonacci series that are even, their
        sum is 10
        """
        self.assertEqual(even_fibsum(20), 10)

File: node/test/problem_02_test.js
var prb = require("../problems_01_25/problem_02.js");
var assert = require("assert");

describe("Problem_02", function() {
    describe("#EvenFibSum()", function() {
        // There are 2 terms (2,8) in the Fib. Seq. 
        //less than 20; their sum is 10
        it("should return 10 when the input is 20", function() {
            assert.equal(10, prb.EvenFibSum(20));
        });
    });
});
